Status update 2022-06-17 (#101)
puppetlabs/content-and-tooling-team: content/blog/updates/2022-06-17-status-update.md

---
author: "david22swan"
categories:
  - updates
date: "2022-06-17"
blog_tags: null
title: "2022-06-17: Content & Tooling Team Status Update"
---

## We Have a New Member

As a start to this update I would like to officially welcome Claire McDyre to the CAT team!!!

> She's taking over as our PM and is certain to have some good ideas on how the team is going to move in the future and I'm sure that all of you are just as interested as e are in seeing them come to fruition.

Reviewer: ```suggestion
She's taking over as our PM and is certain to have some good ideas on how the team is going to move in the future and I'm sure that all of you are just as interested as we are in seeing them come to fruition.
```

## VCSRepo

> On a more technical side some of you may have noticed that a goo amount of work has been put into the VCSRepo module by our own [Craig][craig], who has been working to get it back into a good condition and ensure that once again matches the high standards that we hold.

Reviewer: ```suggestion
On a more technical side some of you may have noticed that a good amount of work has been put into the VCSRepo module by our own [Craig][craig], who has been working to get it back into a good condition and ensure that once again matches the high standards that we hold.
```

## Apache

> Similar to the news stated above, you may have also noticed a large amount of activity on the Apache module these last few weeks, kicked of by [my][me] own work to properly enforce the syntax rules espoused by the community.

Reviewer: ```suggestion
Similar to the news stated above, you may have also noticed a large amount of activity on the Apache module these last few weeks, kicked of by [my][me] to properly enforce the syntax rules espoused by the community.
```

> Ensuring the emoval of any remanining legacy facts and assigning types and descriptions to each and every variable within the module.

Reviewer: ```suggestion
Ensuring the removal of any remaining legacy facts and assigning types and descriptions to each and every variable within the module.
```

> This work has been at first assisted and then continued on by several members of the community who have all either helped correct errors that I have made in the process and even worked to further improve upon the changes made, so to [ekohl][ekohl], [chillenger][chillenger], [traylenator][traylenator] and [cocker-cc][cocker-cc] I would just like to say thank you!

Reviewer: ```suggestion
This work has been at first assisted and then continued on by several members of the community who have all either helped with corrections or worked to further improve upon the changes made, so to [ekohl][ekohl], [chillenger][chillenger], [traylenator][traylenator] and [cocker-cc][cocker-cc] I would just like to say thank you!
```

## Redhat Enterprise Linux 9

Finally to round up our technical news I am officially announcing support for Redhat 9 on our Modules.
While there are a few left that still need some love to get the associated PRs merged we are now ready and able to help with any issues that may arise from the OS and our Modules.

## That's All

Anyway, that's everything.

> So have a great weekend from everyone here at the modules team, but

Reviewer: ```suggestion
So have a great weekend from everyone here in the CAT team!
```
